### PR TITLE
 Convert host names and domains to lower case in DB replies

### PR DESCRIPTION
**Pull request by DL6ER**
**By submitting this pull request, I confirm the following:** 

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md), as well as this entire template.
- [X] I have made only one major change in my proposed changes.
- [X] I have commented my proposed changes within the code.
- [X] I have tested my proposed changes.
- [X] I am willing to help maintain this change if there are issues with it later.
- [X] I give this submission freely and claim no ownership.
- [X] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [X] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [X] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#git-commit---signoff))

---

**What does this PR aim to accomplish?:**

Compensate for the fact that new data is stored with lower-case hosts and domains in the database, while earlier data may be still mixed case.

**How does this PR accomplish the above?:**

Query data and unify the result through converting everything the lower case.

### CODE REVIEW

File: api_db.php [hunk 1]
```diff
@@ -94,15 +94,28 @@ function SQLite3_connect($trytoreconnect)
...
+			{
+				// Entry already exists, add to it (might appear multiple times due to mixed capitalization in the database)
+				$clients[$c] += intval($row[1]);
+			}
+			else
+			{
+				// Entry does not yet exist
+				$clients[$c] = intval($row[1]);
+				// Increase number of clients
+				$num++;
```
Comment (Mcat12) [inline api_db.php]: We should go through every returned result and condense, then sort, then limit to 10. If we limit first, then we could have even more inaccurate results.

File: api_db.php [hunk 2]
```diff
@@ -124,14 +137,28 @@ function SQLite3_connect($trytoreconnect)
...
+			{
+				// Entry already exists, add to it (might appear multiple times due to mixed capitalization in the database)
+				$domains[$c] += intval($row[1]);
+			}
+			else
+			{
+				// Entry does not yet exist
+				$domains[$c] = intval($row[1]);
+				// Increase number of domains
+				$num++;
```
Comment (Mcat12) [inline api_db.php]: See above.

File: api_db.php [hunk 3]
```diff
@@ -94,16 +94,33 @@ function SQLite3_connect($trytoreconnect)
...
+				// Entry does not yet exist
+				$clients[$c] = intval($row[1]);
+			}
 		}
+
+	// Sort by number of hits
+	arsort($domains);
+
+	// Extract only the first ten entries
+	$domains = array_slice($domains, 0, 10);
```
Comment (Mcat12) [inline api_db.php]: Copy-paste error. Should be operating on `$clients`

Comment (DL6ER) [reply api_db.php]: No, it shouldn't?

Comment (Mcat12) [reply api_db.php]: Probably was a force-push

**Pull request merged**